webextensions/common/common.js

```diff
@@ -334,6 +337,41 @@ configs.$loaded.then(() => {
 });
 
 
+export function filterMap(arr, callback, thisArg) {
```

Comment:
Could you use `array` or `items` instead of a shortened name `arr`, to uniform the coding style?